src/connection/Connection.ts

```diff
@@ -406,6 +419,13 @@ export class Connection {
 
         return relationMetadata.junctionEntityMetadata;
     }
+    
+    /**
+     * Creates an Entity Manager for the current connection with the help of the EntityManagerFactory.
+     */
+    createEntityManager(queryRunner?: QueryRunner): EntityManager {
```

Comment:
This is the only problem left....I don't know a way around making it public, because I need it in `EntityManager` to remove the dependency to `EntityManagerFactory` which again depends on `EntityManager`. I don't know why this error appeared now, after a added a second special EntityManager...

Reply:
Okay, lets left it then